SECURITY.md

```diff
@@ -16,29 +16,93 @@ You may need to [update the security provider](https://developer.android.com/tra
 
 ## TLS with OpenSSL
 
-This is currently the recommended approach for using gRPC over TLS (on non-Android systems). 
+This is currently the recommended approach for using gRPC over TLS (on non-Android systems).
 
-### Benefits of using OpenSSL
+The main benefits of using OpenSSL are:
 
 1. **Speed**: In local testing, we've seen performance improvements of 3x over the JDK. GCM, which is used by the only cipher suite required by the HTTP/2 spec, is 10-500x faster.
 2. **Ciphers**: OpenSSL has its own ciphers and is not dependent on the limitations of the JDK. This allows supporting GCM on Java 7.
 3. **ALPN to NPN Fallback**: if the remote endpoint doesn't support ALPN.
 4. **Version Independence**: does not require using a different library version depending on the JDK update.
 
-### Requirements for using OpenSSL
+Support for OpenSSL is only provided for the Netty transport via [netty-tcnative](https://github.com/netty/netty-tcnative), which is a fork of
+[Apache Tomcat's tcnative](http://tomcat.apache.org/native-doc/), a JNI wrapper around OpenSSL. As of version `1.1.33.Fork13`, netty-tcnative provides two options for usage: statically or dynamically linked. For simplification of setup,
+we recommend using `netty-tcnative-boringssl-static`, which is statically linked against BoringSSL and Apache APR. Using this artifact requires no extra installation and guarantees that ALPN and the ciphers required for
```

Comment:
The difference of setup difficulty varies across platforms. On Linux, as you pointed out, it's simple enough. Windows users don't have the same experience, however. The additional setup required (on non-Linux) has been a major pain point for our users. The goal here is to recommend the simplest approach for getting started using grpc. Agreed that in many production (server-side) setups, which are typically Linux anyway, will likely have an OpenSSL installed that is managed by the system administrator ... and will therefore use the netty-tcnative artifact instead. Our instructions still describe how to use this (below), but we recommend the statically linked version and list it here first to ease the burden on getting started.